admin-dev/themes/new-theme/js/product-page/product-bulk-combinations.js

```diff
@@ -1,39 +1,39 @@
+import $ from 'jquery';
+
 /**
  * Combination bulk actions management
  */
 export default function() {
 
-  var bulkForm = document.querySelector('#bulk-combinations-container');
-  var combinationsTable = document.querySelector('#accordion_combinations');
-  var deleteCombinationsBtn = document.querySelector('#delete-combinations');
-  var applyChangesBtn = document.querySelector('#apply-on-combinations');
-  var selectAllCheckbox = document.querySelector('#toggle-all-combinations');
+  var bulkForm = $('#bulk-combinations-container');
+  var combinationsTable = $('#accordion_combinations');
+  var deleteCombinationsBtn = $('#delete-combinations');
+  var applyChangesBtn = $('#apply-on-combinations');
 
   return {
     'init': function init() {
       var that = this;
       // stop propagation on buttons
-      deleteCombinationsBtn.addEventListener('click', (event) => {
+      deleteCombinationsBtn.on('click', (event) => {
         event.preventDefault();
         that.deleteCombinations();
```

Comment:
I'm not sure at all, but can't you use `this` instead of `that`, since you are using `=>` ?